src/workers/postBannedRep.ts

```diff
@@ -17,7 +17,10 @@ const worker: Worker = {
   handler: async (message, con, logger): Promise<void> => {
     const data: Data = messageToJson(message);
     const { id, authorId, scoutId, flags } = data.post;
-    const { deletedBy } = flags;
+    const parsedFlags =
+      typeof flags === 'string' ? JSON.parse(flags as string) : flags;
+    const { deletedBy } = parsedFlags;
```

Comment:
Oh, I bet because in the tests, we send and actual object to the mock handler, the tests was successful. But in the real word, it gets sent as a string that gets parsed 😬

Reply:
Yep, wasn't sure if there was some case where a topic could send it as parsed, so this felt safer 😅